modal.css

```diff
@@ -0,0 +1,158 @@
+.subtittle {
+  margin-right: 2%;
+  float: left;
+}
+
+h2.popUp {
+  margin-top: 10%;
+  margin-bottom: 10%;
+  width: 50;
+}
+
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 95%;
+  z-index: 2;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  background: #fff;
+  justify-content: start;
+  margin-right: 3%;
+  margin-left: 3%;
+  margin-top: 4%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+}
+
+.projectPopup {
+  display: grid;
+  height: 100%;
+  position: fixed;
+  padding: 0;
+  margin: 0;
+  background-color: #020a58;
+}
+
+.uil-multiply {
+  float: right;
+}
+
+.popupLinkContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  justify-content: start;
+  align-items: start;
+  width: 100%;
+}
+
+.popupLinkContainer a {
+  margin: 4%;
+  color: #396df2;
+  padding: 0.5rem 0.5rem;
+  text-align: center;
+  cursor: pointer;
+  border: 1px solid #396df2;
+  border-radius: 9px;
+}
+
+.popupLinkContainer a:focus {
+  background-color: #020a58;
+}
+
+.popupLinkContainer a:hover {
+  background-color: #020a58;
+  color: white;
+  transition: 0.2s;
+}
+
+.popupLinkContainer a:disabled {
+  background-color: #f4f1f0;
+  border: none;
+  color: #5b5957;
+}
+
+#closeBtn {
+  font-family: 'Roboto', sans-serif;
+  font-size: 2rem;
+  position: absolute;
+  cursor: pointer;
+  transition: 0.1s;
+  float: right;
+  top: 2%;
+  right: 10%;
+}
+
+#closeBtn:hover {
+  scale: 1.3;
+  font-weight: 700;
+}
+
+.btnIcons {
+  max-width: 60%;
+  height: auto;
+}
+
+.sublinks > ul {
+  padding: 0;
+  display: flex;
+  width: 50%;
+  justify-content: space-around;
+  flex-direction: row;
+  list-style-type: disc;
+  color: #000;
+  margin-left: 3%;
+  margin-bottom: 2%;
+  direction: ltr;
+}
+
+.sublinks > ul > li {
+  list-style-type: disc;
+  left: 5%;
+  color: #7a869a;
+}
+
+@media (min-width: 768px) {
+  .popupContainer {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Good job adding the popup but it seems like some of the text is getting clipped as shown below, so please make sure to add a scroll feature for a better user experience. ✔
- Your design:
![5oI8FKYSjv](https://user-images.githubusercontent.com/104968236/233389655-56b51a4b-b3b3-4205-9062-1ad99e4ea80b.png)
- Figma design:
  ![image](https://user-images.githubusercontent.com/104968236/233390040-a70e63f2-6549-447c-a4d4-1cb36f4a3968.png)

- [ ] I've also noticed the spacing in the elements is bit disproportionate and not as shown in the Figma design so please make sure you fix that as well I've tried to highlight some of the visible ones below:
  ![Inked5oI8FKYSjv](https://user-images.githubusercontent.com/104968236/233391768-e739ea11-021a-4d40-984f-16f492c9b3a8.jpg)